plugin.video.psvue/resources/lib/ps_vue.py

```diff
@@ -428,12 +661,19 @@ def get_stream(url, airing_id, channel_id, program_id, series_id, tms_id, title,
     }
 
     r = requests.get(url, headers=headers, cookies=load_cookies(), verify=VERIFY)
-    json_source = r.json()
+
+    try:
+        json_source = r.json()
+    except ValueError:
+        xbmcgui.Dialog().ok('OOPSIES', 'This episode of the', title, 'Has not aired yet')
```

Comment:
can you please use localized strings here, its a message shown to users

Reply:
Sure, no problem.